.gitignore

```diff
@@ -1 +1,5 @@
-node_modules/
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
![image](https://github.com/mryeminthu/To-Do-list/assets/93039359/d09728b0-cd4f-4151-8205-de346e807843)

- [ ] It would be nice if you fix the link shown in the image above from your Table of Contents in your readme file :thinking: because it is broken right now :fearful: Kindly fix it or remove it **(If you don't have any particular section FAQ)** to make your readme descriptive and professional :+1: